test/starlark_tests/ios_extension_tests.bzl

```diff
@@ -259,6 +259,17 @@ def ios_extension_test_suite(name):
         tags = [name],
     )
 
+    # Test ext with resources bundles them at the top level
+    archive_contents_test(
+        name = "{}_contains_resources_test".format(name),
+        build_type = "simulator",
+        target_under_test = "//test/starlark_tests/targets_under_test/ios:ext_with_resources",
+        contains = [
+            "$BUNDLE_ROOT/additional.txt",
```

Comment:
awesome test

Reply:
haha I figured I would avoid adding a new fixture if I didn't need to

Reply:
Yea I was being serious not snarky!